1 Table.sql

```diff
@@ -0,0 +1,75 @@
+use confectioneryDB
+go
+drop table if exists dbo.Baking
+go
+create table dbo.baking(
+    BakingID int not null identity primary key,
+
+    CustomerName varchar(100) not null 
+        constraint ck_Baking_customer_name_cannot_be_blank check (CustomerName <> ''),
+
+    StoreBranch varchar (20) not null
+        constraint ck_Baking_store_branch_must_be_lakewood_or_brooklyn check (StoreBranch in ('lakewood', 'brooklyn')),
+
+    OrderDate date not null 
+        constraint ck_Baking_order_date_must_be_after_1_1_2021 check (OrderDate >= '1/1/2021'),
+
+    BaseType varchar(50) not null 
+        constraint ck_Baking_base_type_must_be_eithe_choc_vanilla_coconut_chocpeanutbutter_banana_strawberryshortcake_or_sugar
+            check(BaseType in ('Chocolate', 'Vanilla', 'Coconut', 'Chocolate Peanut Butter', 'Banana', 'Strawberry shortcake', 'sugar')),
+
+    ItemType varchar(30) not null
+        constraint ck_Baking_item_type_must_be_either_cake_or_cookie_or_cupcake check(ItemType in ('cake', 'cookie', 'cupcake')),
+    
+    Topping varchar(50) not null
+        constraint ck_Baking_topping_must_be_either_icing_fondant_choc_caramel_strawberry_coconut_peanut_butter_vanilla_or_no_topping
+            check (Topping in ('Royal icing', 'fondant', 'frosting', 'chocolate', 'caramel', 'strawberry', 'coconut', 'peanut butter', 'vanilla', 'no topping')),
+
+    Picture bit not null,
+
+    OrderSpecifics varchar(100),
```

Comment:
The column OrderSpecifics is not marked as not null. Even though we might leave it blank, we don't need it to be null.
The picture column is fine, my mistake.

Reply:
Yes, you can close it.